### PR TITLE
GIX-1789: New colors

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

We want to align the colors in Figma and the NNS Dapp.

In this PR, we use the colors used in Figma for the theme variables.

More work is needed to align, clean and improve the variable colors.

# Changes

* Secondary button doesn't need the trick to have a border with gradient. Therefore, we remove the extra complexity because we can use a transparent background.
* Add three new primitive colors.
* Change all primitive color values in the theme files light.scss and dark.scss to use the new colors.

# Screenshots

NNS Dapp deployed here https://kbruq-4yaaa-aaaaa-aacta-cai.nnsdapp.dfinity.network/ for testing.

### CODE REVIEW

File: src/lib/components/Toast.svelte [hunk 1]
```diff
@@ -27,7 +27,7 @@
       ["error"]: IconError,
       ["info"]: IconInfo,
       ["custom"]: undefined,
-    }[level]);
+    })[level];
```
Comment (peterpeterparker) [inline src/lib/components/Toast.svelte:30]: is this correct? not sure how it relates to the PR

File: src/lib/components/Toast.svelte [hunk 2]
```diff
@@ -38,6 +38,7 @@
   let overflow: "scroll" | "truncate" | "clamp" | undefined;
   let position: ToastPosition | undefined;
   let icon: typeof SvelteComponent | undefined;
+  // TODO-colors: Can we remove this?
```
Comment (peterpeterparker) [inline src/lib/components/Toast.svelte:41]: probably no because of the toast top right that sometimes has to be forced in a particular theme

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

Added two minor comments if you want to have a look before merge.

File: src/lib/components/Toast.svelte (hunk 2 above)
Comment (lmuntaner) [reply src/lib/components/Toast.svelte:41]: thanks for the tip!

File: src/lib/components/Toast.svelte (hunk 1 above)
Comment (lmuntaner) [reply src/lib/components/Toast.svelte:30]: It was my lint that changed it.

**Pull request merged**